WcaOnRails/app/controllers/server_status_controller.rb

```diff
@@ -88,43 +87,6 @@ def label
   end
 end
 
-class CertificateCheck < StatusCheck
```

Comment:
Just an idea, not strictly necessary: Can we implement a certificate check through some Amazon APIs? Just as a double security layer...

Reply:
Okay, nevermind then :upside_down_face: